sources/views/details.js

```diff
@@ -0,0 +1,91 @@
+import {JetView} from "webix-jet";
+import {contacts} from "../models/contacts";
+import {statuses} from "../models/statuses";
+
+export default class Details extends JetView {
+	config() {
+		const details = {
+			view: "template",
+			localId: "userDetails",
+			template: contact => `
+				<div class="details_wrapper">
+					<div class="details_header">
+						<h2>${contact.FirstName || "-"} ${contact.LastName || "-"}</h2>
+					</div>
+					<div class="details_body">
+						<div class="details_image">
+							<image src="${contact.Photo || "https://ru.seaicons.com/wp-content/uploads/2015/06/Users-User-Male-4-icon.png"}" />
+							<p class="status">${contact.Status || "-"}</p>
+						</div>
+						<div class="details_contacts">
+							<span class="mdi mdi-email"> ${contact.Email || "-"}</span>
+							<span class="mdi mdi-skype"> ${contact.Skype || "-"}</span>
+							<span class="mdi mdi-tag"> ${contact.Job || "-"}</span>
+							<span class='mdi mdi-briefcase'> ${contact.Company || "-"}</span>
+						</div>
+						<div class="details_contacts">
+							<span class='mdi mdi-calendar-month'> ${contact.Birthday || "-"}</span>
+							<span class='mdi mdi-map-marker'> ${contact.Address || "-"}</span>
+					</div>
+					</div>
+				</div>
+			`
+		};
+
+		const buttons = {
```

Comment:
From step 1 description
Required features:
on this step, buttons Delete and Edit are inactive

Reply:
Made them inactive by adding disabled property.